components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -181,4 +181,10 @@
     <string name="mozac_feature_addons_updater_dialog_last_attempt">Last attempt:</string>
     <!-- Displayed in the "Status" field for the updater when an add-on has been correctly updated -->
     <string name="mozac_feature_addons_updater_dialog_status">Status:</string>
-</resources>
\ No newline at end of file
+    <!-- Text shown in he dialog when add-on installation is completed. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_installed_dialog_title">%1$s has been added to Firefox</string>
```

Comment:
nit: We could add a variable instead of "Firefox" and replace it with the [appName](https://github.com/mozilla-mobile/android-components/blob/798ee8f9a3786739859181d27fbb81674fa7be87/components/support/ktx/src/main/java/mozilla/components/support/ktx/android/content/Context.kt#L42)

Reply:
Nice